.github/workflows/udeps.yml

```
@@ -115,7 +115,7 @@ jobs:
           if [ "${UTILS}" == "true" ]; then crates[${#crates[@]}]="\"./core/tauri-utils\""; fi
           if [ "${BUNDLER}" == "true" ]; then crates[${#crates[@]}]="\"./tooling/bundler\""; fi
           if [ "${CLI}" == "true" ]; then crates[${#crates[@]}]="\"./tooling/cli\""; fi
-          echo ::set-output name=matrix::[$crates]
+          echo "matrix=[$crates]" >> "$GITHUB_ENV"
```

I should've noticed this earlier, we should be using `$GITHUB_OUTPUT`
```suggestion
          echo "matrix=[$crates]" >> "$GITHUB_OUTPUT"
```